src/Core/Auth/Services/Implementations/AuthRequestService.cs

```diff
@@ -297,10 +297,34 @@ private async Task NotifyAdminsOfDeviceApprovalRequestAsync(OrganizationUser org
             return;
         }
 
-        var admins = await _organizationUserRepository.GetManyByMinimumRoleAsync(
+        var adminEmails = await GetAdminAndAccountRecoveryEmailsAsync(organizationUser.OrganizationId);
```

Comment:
None-blocking question: I guess it's safe to say that `adminEmails` will never be empty because an organization will have at least one owner?

Reply:
Yes, exactly! We always check if an organization has at least one confirmed owner when updating/removing a user.